generator/processors/Microsoft.Compute.ts

```diff
@@ -60,9 +60,12 @@ export const postProcessor: SchemaPostProcessor = async (namespace, apiVersion,
     if (schema.resourceDefinitions?.cloudServices_updateDomains) {
         delete schema.resourceDefinitions.cloudServices_updateDomains;
     }
-    if (schema.resourceDefinitions?.cloudServices?.resources) {
+    if (schema.resourceDefinitions?.cloudServices?.properties?.resources) {
         delete schema.resourceDefinitions.cloudServices.resources
     }
+    if (schema.cloudServices_updateDomains_childResource) {
+        delete schema.cloudServices_updateDomains_childResource
+    }
```

Comment:
I believe this should be `schema.definitions?.cloudServices_updateDomains_childResource`

Note that if you're using VSCode, you can find the JSON path of the cursor by looking at the breadcrumbs at the top of the editor:
![image](https://user-images.githubusercontent.com/38542602/174623117-93594a03-ac24-4c1b-89bc-b748a22f676f.png)

Reply:
No worries. I've merged. I suggest just keeping an eye on the next generation pipeline run (should be ~24h from now) and making sure it has the intended effect